From f621ee23d9f304bd0d36727d35afcae90ff357ad Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=A9r=C3=A9my=20Bobbio?= <lunar@debian.org>
Date: Sun, 28 Dec 2014 16:21:46 +0000
Subject: [PATCH 2/3] dh_icons: output icon list in a stable order

We now sort the icon list in order to guarantee a stable output.
This was preventing some packages to build reproducibly.
---
 dh_icons | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dh_icons b/dh_icons
index 916f188..b506f1f 100755
--- a/dh_icons
+++ b/dh_icons
@@ -61,7 +61,7 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 			push @dirlist, "$baseicondir/$subdir" if $needs_cache;
 		}
 		if (@dirlist and ! $dh{NOSCRIPTS}) {
-			my $list=join(" ", @dirlist);
+			my $list=join(" ", sort @dirlist);
 			autoscript($package,"postinst","postinst-icons","s%#DIRLIST#%$list%");
 			autoscript($package,"postrm","postrm-icons","s%#DIRLIST#%$list%");
 		}
-- 
2.1.3

